### PR TITLE
use PostCSS 4.1 API

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 1.2.0 - 2015-05-23
```
Comment (MoOx) [inline CHANGELOG.md]: This should be 2.0 since you are introducing a breaking change for pre 4.1 users. Am I right ?

Comment (ai) [inline CHANGELOG.md]: Nope, `postcss.plugin` generates compatible API for previous PostCSS.

Comment (MoOx) [inline CHANGELOG.md]: Hummm... Awesome then ! This is good to know and great !

**Comment by magsout** [comment]
:+1:  thanks

**Pull request merged**